web/styles/popovers.css

```diff
@@ -161,33 +147,44 @@
     }
 }
 
-.user-card-popover-actions .custom_user_field {
+.custom-profile-field-value {
     display: flex;
     align-items: center;
+    gap: 5px;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
+}
 
-    .value {
-        text-overflow: ellipsis;
-        overflow: hidden;
-        white-space: nowrap;
-    }
-
+.custom-profile-field-value.rendered_markdown p {
+    text-overflow: ellipsis;
+    overflow: hidden;
+    white-space: nowrap;
     /* Overriding CSS from rendered_markdown.css */
-    .rendered_markdown p {
-        margin: 0;
+    margin: 0;
+}
+
+.custom-profile-field-link {
+    color: var(--color-text-item);
+
+    &:hover,
+    &:focus {
+        color: var(--color-text-link);
+        text-decoration: none;
     }
 
-    /* Overriding CSS from bootstrap > nav-list. */
-    .custom-profile-fields-link {
-        padding-top: 0;
-        padding-bottom: 0;
+    .custom-profile-field-text {
+        text-overflow: ellipsis;
+        overflow: hidden;
+        white-space: nowrap;
 
         &:hover {
-            background-color: transparent !important;
+            text-decoration: underline;
         }
+    }
 
-        &:focus {
-            background-color: transparent !important;
-        }
+    ~ i {
+        margin-left: auto;
```

Comment:
We use `margin-left: auto` on the copy icon instead of `flex-grow: 1` on
the link item, since we don't want the underline to span across the whole
width of the element.